pkgs/applications/networking/instant-messengers/cinny/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "cinny";
+  version = "1.2.0";
+
+  src = fetchurl {
+    url = "https://github.com/ajbura/cinny/releases/download/v${version}/cinny-v${version}.tar.gz";
+    sha256 = "sha256-eQKi6SawmPJecOrueCkarsvqpNvXoMx2cHQ6/vUpUJs=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/
+    cp -R . $out/
+
+    runHook postInstall
+  '';
+
+  meta = {
```

Comment:
Use with lib; here to avoid the lib. noise further down

Reply:
Done. Thanks!